django/chat/templates/chat/components/chat_list_item_title_edit.html

```diff
@@ -22,12 +22,15 @@
 
 {% block page_script %}
     <script>
-        htmx.onLoad((element) => {            
-            if(element instanceof HTMLFormElement) {                
+        htmx.onLoad((element) => {
+            if(element instanceof HTMLFormElement) {
                 const input = document.getElementById('{{ form.title.id_for_label }}');                
-                input.setSelectionRange(-1, -1);                            
```

Comment:
This was broken, and is addressed better with onfocus in the input element (see forms.py)